tezos/new_context/src/ffi.rs

```diff
@@ -540,14 +538,23 @@ ocaml_export! {
         rt,
         context: OCamlRef<DynBox<TezedgeContextFFI>>,
         value: OCamlRef<OCamlBytes>
-    ) -> OCaml<DynBox<WorkingTreeFFI>> {
+    ) -> OCaml<Result<DynBox<WorkingTreeFFI>, String>> {
```

Comment:
Ok, before merging I have to update this on the OCaml side, almost forgot.

Reply:
Ah yes, thanks